### PR TITLE
Add LNURL-pay mocking

### PR DESCRIPTION
**Pull request by gcomte**

### CODE REVIEW

File: mock/breez-sdk/src/lib.rs [hunk 1]
```diff
@@ -648,9 +696,23 @@ pub async fn parse(input: &str) -> Result<InputType> {
         return Ok(Bolt11 { invoice });
     }
 
-    // If the input is not a Bolt11, we just assume it is a LNURL-withdraw request.
-    // This is being used for topups, LNURL-pay is less used.
     // Without requesting the server, it is not possible to know whether an LNURL string is a pay or withdraw request
+    // So instead we interpret the string 'lnurlp' as if it was an LNUrL-pay string
+    if input == "lnurlp" {
+        println!("Returning LnUrlPay");
```
Comment (danielgranhao) [inline mock/breez-sdk/src/lib.rs:702]:
```suggestion
        debug!("Returning LnUrlPay");
```

File: mock/breez-sdk/src/lib.rs [hunk 2]
```diff
@@ -648,9 +696,23 @@ pub async fn parse(input: &str) -> Result<InputType> {
...
+        return Ok(InputType::LnUrlPay {
+            data: LnUrlPayRequestData {
+                callback: "https://lnurl.dummy.com/lnurl-pay/callback/e9a0f330f34ac16d297094f568060d267bac6319a7f0d06eaf89d7fc1512f39a".to_string(),
+                min_sendable: 0,
```
Comment (danielgranhao) [inline mock/breez-sdk/src/lib.rs:706]: Min min_sendable is 1 according to spec
```suggestion
                min_sendable: 1,
```

File: mock/breez-sdk/src/lib.rs (hunk 1 above)
Comment (gcomte) [reply mock/breez-sdk/src/lib.rs:702]: Oh sorry, that was only for development purposes. I'm gonna remove it, I don't think there's much value in writing this to the log.

Comment (gcomte) [reply mock/breez-sdk/src/lib.rs]: 94f3cc6597d07becb26f658c5cb93ff95fcef133

**Review by danielgranhao — APPROVED** [review]

**Pull request merged**